src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -343,22 +342,17 @@ private void addScmDetails() {
                 factsBuilder.addCulprits(run.getResult(), build.getCulprits());
 
                 ChangeLogSet changeSet = build.getChangeSet();
-                List<ChangeLogSet.Entry> entries = new LinkedList<>();
                 Set<ChangeLogSet.AffectedFile> files = new HashSet<>();
+                Set<User> authors = new HashSet<>();
+
                 for (Object o : changeSet.getItems()) {
                     ChangeLogSet.Entry entry = (ChangeLogSet.Entry) o;
-                    entries.add(entry);
+                    authors.add(entry.getAuthor());
                     files.addAll(getAffectedFiles(entry));
                 }
-                if (!entries.isEmpty()) {
-                    Set<User> authors = new HashSet<>();
-                    for (ChangeLogSet.Entry entry : entries) {
-                        authors.add(entry.getAuthor());
-                    }
 
-                    factsBuilder.addDevelopers(authors);
-                    factsBuilder.addNumberOfFilesChanged(files);
-                }
+                factsBuilder.addDevelopers(authors);
```

Comment:
can be passed without ``isEmpty`` as this is already validated in ``addDevelopers``